odk1-src/central-install.rst

```diff
@@ -10,9 +10,9 @@ Unlike ODK Aggregate, Central does not run on Google App Engine. Instead, it is
 Using the Sandbox
 -----------------
 
-If you only want to try ODK Central to see if it's suitable for your project, consider skipping installation altogether and using the `Sandbox installation <https://sandbox.central.opendatakit.org/>`_ we've provided for exactly this reason. Do note that since there is only one Sandbox, all Sandbox users will be able to see each others' email addresses, form definitions, and submission data, so please be careful if you have sensitive information you wish to keep secret.
+If you only want to try ODK Central to see if it's suitable for your project, consider skipping installation altogether and using the `Sandbox installation <https://sandbox.central.getodk.org/>`_ we've provided for exactly this reason. Do note that since there is only one Sandbox, all Sandbox users will be able to see each others' email addresses, form definitions, and submission data, so please be careful if you have sensitive information you wish to keep secret.
```

Comment:
Is moving the Central and Aggregate sandboxes part of the migration checklist?

Reply:
No, I figured it's something that can also be done afterwards. So this is a mistake. I'll move it back. Unless you think it can/should be done?

Reply:
I would be nice to do soon after at least, I think.

Reply:
I think it can be done concurrently. I can't imagine it will be difficult and I think @yanokwa plans to just batch all these kinds of changes. So I think this can stay.

Reply:
Oh, I see you already switched it back. Doesn't matter either way as long as the state is tracked somewhere. Is there a master checklist?

Reply:
I've added it here: https://github.com/opendatakit/docs/issues/1208